Tasks/AzureResourceGroupDeploymentV2/operations/ResourceGroup.ts

```diff
@@ -181,7 +181,14 @@ export class ResourceGroup {
                         policyLink = this.getPolicyHelpLink(error.details[i]);
                         errorMessage = this.getPolicyErrorMessage(error.details[i]);
                     } else {
-                        errorMessage = util.format("%s: %s %s", error.details[i].code, error.details[i].message, error.details[i].details);
+                        errorMessage = util.format("%s: %s", error.details[i].code, error.details[i].message);
+                        if(error.details[i].details) {
```

Comment:
do we really need if-else condition. can we not just do JSON.stringify in original line itself

`errorMessage = util.format("%s: %s %s", error.details[i].code, error.details[i].message, JSON.stringify(error.details[i].details));`

The only downside is that if details is string object then it will have two double quotes.

Reply:
I'll keep the if condition, otherwise undefined can come in error logs, which could confuse